src/Microsoft.DotNet.Interactive.FSharp/FSharpKernel.fs

```diff
@@ -220,7 +220,7 @@ type FSharpKernel () as this =
                 match result with
                 | Some(value) when value.ReflectionType <> typeof<unit> ->
                     let value = value.ReflectionValue
-                    let formattedValues = FormattedValue.FromObject(value)
+                    let formattedValues = FormattedValue.CreateManyFromObject(value)
                     context.Publish(ReturnValueProduced(value, codeSubmission, formattedValues))
```

Comment:
minor: Considering that a single formatted value is probably the common case, I wonder if event constructors (like `ReturnValueProduced()`) should have overloads that accept a single formatted value.

Reply:
Never mind - I realize that a single formatted value is probably not the common case since it depends on the set of supported mime types associated with the type of the value...